static/docs/commands-reference/import-url.md

```diff
@@ -95,13 +95,9 @@ Both methods generate an equivalent [stage file](/doc/commands-reference/run)
 user from having to manually copy files from each of the remote storage schemes,
 and from having to install CLI tools for each service.
 
-Note that import stages are locked by default. Use `dvc update` manually on them
-to force updating the downloaded file or directory from the external data
-source.
-
-> If a stage is unlocked (editing the `lock` value in its DVC-file, for example
-> using `dvc unlock`), they will start to be checked by `dvc status`, and
-> updated by `dvc repro`.
+Note that import stages are considered always locked. (See `dvc lock`.) They can
```

Comment:
let's remove `See dvc lock` and that they cannot be unlocked. It will be a different type of lock. 

Something like this:

Note that import stages are considered always "locked" - it means that if you run `dvc repro` DVC won't try to update them. You must run `dvc update` explicitly ...

Reply:
the same for import

Reply:
Same for `update`. Updated in 5f14b34